content/sensu-go/6.2/api/_index.md

```diff
@@ -45,7 +45,11 @@ See the [RBAC reference][3] for more information about configuring Sensu users a
 ## Data format
 
 The Sensu API uses JSON-formatted requests and responses.
-In terms of [sensuctl output types][1], the Sensu API uses the `json` format, not `wrapped-json`.
+In terms of output formats, for `core`-group API responses, Sensu uses `json` format.
+For [commercial][8] APIs, Sensu uses `wrapped-json` format for API responses.
```

Comment:
To clarify the APIs that are specific to commercial or by having a commercially build and licensed versions all calls to either sets of APIs? I think its the former based on how this reads but I wanted to double check.